nav2_navfn_planner/src/navfn_planner.cpp

```diff
@@ -124,18 +134,18 @@ NavfnPlanner::computePathToPose(const nav2_tasks::ComputePathToPoseCommand::Shar
     // Publish the plan for visualization purposes
     RCLCPP_INFO(get_logger(), "Publishing the valid path.");
     publishPlan(result);
-    publishEndpoints(command);
+    publishEndpoints(start->pose.pose, command->pose);
 
     // TODO(orduno): Enable potential visualization
 
     RCLCPP_INFO(get_logger(),
       "Successfully navigated to (%.2f, %.2f) with tolerance %.2f",
```

Comment:
This one should probably be RCLCPP_DEBUG as well

Reply:
@crdelsey Yes, I agree. I'll update.